tests/Backend/CommonPart2/SlicedImportsTest.php

```diff
@@ -62,7 +62,6 @@ public function testSlicedImportSingleFile(): void
     {
         $this->skipTestForBackend([
             self::BACKEND_TERADATA,
-            self::BACKEND_BIGQUERY,
```

Comment:
toto ale bude pravda az s updatem driveru ne? a to v tom PR connectino se nedeje

Reply:
to píšu v mojém revjů :)

Reply:
imo @martinjunger referencuje špatné PR v connection ten multiple test pojede až s https://github.com/keboola/connection/pull/4148

Reply:
a jo on tu multiple test není :facepalm: pletu to já :)

Reply:
smazal sem svůj commit

Reply:
jo, sorry za zmatky, ale tady jsem resil primarne aby ty testy jeli s novyma parametrama - skutecny povoleni testu je az tady https://github.com/keboola/storage-api-php-client/pull/972